apps/script-service/Controllers/RunScriptRequest.cs

```diff
@@ -16,4 +17,8 @@ public class RunScriptRequest
 
   [JsonPropertyName("correlationId")]
   public string? CorrelationId { get; set; }
+
```

Comment:
You should reuse the same request body POCO in separate endpoints unless they're actually identical. Define a new POC instead for the request values.

Also the test run shouldn't require a full 'definition'. The consumer of the API should be able to provide just the script as a string in addition to the input values.

Reply:
Hi @tzuge, thank you for your review. I've created another request schema with minimum attributes in it.